### ISSUE TITLE
code release for GT occupancy and flow annotation

### ISSUE THREAD

**Issue by Toytiny** [OpenDriveLab/OccNet#13]
Great work! When do you plan to release your code for GT occupancy and flow annotations?  Or, do you know any other open-sourced GT occupancy generation codes? We'd like to transfer other datasets in a similar way.

**Comment by hli2020** [comment]
Nope. As we plan to host the Challenge in 2024 with some modifications, releasing the procedure of generating annotations is definitely bummer no. You are welcome to contribute! Thanks

**Comment by Toytiny** [comment]
Thanks!

**Comment by rlczddl** [comment]
Hello, it's 2025, I don't know if this part of the code can be open sourced.
